src/main/fc/cli.c

```diff
@@ -152,7 +153,7 @@ static const char * const featureNames[] = {
     "", "TELEMETRY", "CURRENT_METER", "3D", "RX_PARALLEL_PWM",
     "RX_MSP", "RSSI_ADC", "LED_STRIP", "DASHBOARD", "",
     "BLACKBOX", "CHANNEL_FORWARDING", "TRANSPONDER", "AIRMODE",
-    "SUPEREXPO", "VTX", "RX_SPI", "SOFTSPI", "PWM_SERVO_DRIVER", "PWM_OUTPUT_ENABLE", "OSD", NULL
+    "SUPEREXPO", "VTX", "RX_SPI", "SOFTSPI", "PWM_SERVO_DRIVER", "PWM_OUTPUT_ENABLE", "OSD", "RX_ELERES", NULL
```

Comment:
You don't need IRQ check in a separate task - the framework already provides for that. `rxUpdateCheck` in `rx.c` is called regularly. This in turn calls `rxRuntimeConfig.rcFrameStatusFn()` a function provided by the receiver implementation. For SPI this is `rxSpiFrameStatus` in `rx_spi.c`, which in turn calls the function pointer `protocolDataReceived` which is the thing that you should provide and set up. See `rxSpiSetProtocol` for how other protocols set this up.